optimize.sh

```diff
@@ -1,4 +1,6 @@
-#!/bin/sh
+#!/bin/ash
```

Comment:
If it's just for silencing `shellcheck`, the line below will do, and we could keep `sh` here. Better to be compatible between execution and linting, though. So, I guess it's better to switch to `ash`.

Reply:
Using sh has the problem that (in the general case) we cannot use many things we use. Since we know we are on alpine, I wanted to make the shell explicit.